iwf/client.py

```diff
@@ -94,10 +95,20 @@ def start_workflow(
             wf_type, wf_id, starting_state_id, timeout_seconds, input, unreg_opts
         )
 
+    @warnings.deprecated("use wait_for_workflow_completion instead")
     def get_simple_workflow_result_with_wait(
         self,
         workflow_id: str,
         type_hint: Optional[Type[T]] = None,
+    ) -> Optional[T]:
+        return self._unregistered_client.get_simple_workflow_result_with_wait(
+            workflow_id, "", type_hint
+        )
+
+    def wait_for_workflow_completion(
```

Comment:
this is also a rename same as in Java SDK.. too lazy to open a ticket for that (🥹 )https://github.com/indeedeng/iwf-java-sdk/blob/main/src/main/java/io/iworkflow/core/Client.java#L339